Jellyfin.Server/Program.cs

```diff
@@ -139,7 +140,14 @@ private static async Task StartServer(IServerApplicationPaths appPaths, StartupO
                 host = Host.CreateDefaultBuilder()
                     .UseConsoleLifetime()
                     .ConfigureServices(services => appHost.Init(services))
-                    .ConfigureWebHostDefaults(webHostBuilder => webHostBuilder.ConfigureWebHostBuilder(appHost, startupConfig, appPaths, _logger))
+                    .ConfigureWebHostDefaults(webHostBuilder =>
+                    {
+                        webHostBuilder.ConfigureWebHostBuilder(appHost, startupConfig, appPaths, _logger);
+                        if (Environment.GetEnvironmentVariable("ASPNETCORE_IIS_PHYSICAL_PATH") is not null)
```

Comment:
Lets change this ENV to `JELLYFIN_ENABLE_IIS` so the user needs to explicitly opt-in to enabling IIS inside of Jellyfin.
```suggestion
                        if (bool.TryParse(Environment.GetEnvironmentVariable("JELLYFIN_ENABLE_IIS"), out var iisEnabled) && iisEnabled)
```